feat: add assets boilerplate generator

theAmit10 opened the pull request:
## Description
This PR adds an automated CLI solution for generating React Native assets folder structures.

### Implementation Details
- New executable script: `utilities/react-native-assets-generator.js`
- Required permissions: `chmod +x utilities/react-native-assets-generator.js`
- Package.json updates:
  ```json
  "bin": {
    "react-native-assets-boilerplate": "./utilities/react-native-assets-generator.js"
  },
  "files": [
    "utilities/"
  ]
  
  
  
  npm install
  node ./utilities/react-native-assets-generator.js ./src react-native-assets

Asadali00000 commented on package.json:
```diff
@@ -4,7 +4,7 @@
   "main": "src/index.js",
   "bin": {
-    "my-boilerplate-generator": "./src/index.js"
+  "my-boilerplate-generator": "./utilities/react-native-assets-generator.js"
```
index.js is the main file which handle all commands  by changing that no cli command will run , please change it back .

Asadali00000 commented on package.json:
```diff
@@ -37,6 +37,7 @@
   "homepage": "https://github.com/Asadali00000/boilerplate-generator-cli#readme",
   "files": [
     "src/",
+     "utilities/",
```
We didn't need to create a utilities folder — the React Native boilerplate code should go inside the existing boilerplateReactNative folder, which has already been created.

There's also an assets folder, but your version is better. You can follow the structure of the existing one I created to understand how it should be organized.

Asadali00000 commented on utilities/react-native-assets-generator.js:
This file should go inside the assets folder — you can check our other folders to see how we're structuring things.

We don’t need a utilities folder for this, and the file is not following the structure we've already established.

Asadali00000 commented:
Make sure you also add the code inside index.js, or just update the existing assets folder located inside the boilerplateReactNative directory.

If you simply modify the existing assets folder instead of creating a new one, you won’t need to change any references elsewhere — that will be much easier.

theAmit10 commented:
@Asadali00000 I've implemented the requested changes:
- Moved the generator to `boilerplateReactNative/assets/`
- Followed the project structure
- Maintained all existing functionality

Should I:
1. Update this PR with new commits, or
2. Create a new PR with the changes?

Ready for your review when convenient.

theAmit10 commented:
@Asadali00000 Added the requested asset folders:
- animations/
- data/ 
- fonts/
- sounds/
- videos/

No code changes - these will be automatically handled by assets.js
